### ISSUE TITLE
Excessive disk usage report (1+GB)

### ISSUE THREAD

**Issue by RamblinWreck77** [mixpanel/mixpanel-swift#584]
Version: 4.0.4

Noticed this in our Xcode Organizer "Reports" section under "Disk"

<img width="649" alt="Screenshot 2023-02-06 at 1 51 20 PM" src="https://user-images.githubusercontent.com/22692130/217059982-6cf22b5d-814c-4d82-841f-2d926e7faf5d.png">

The insight seemed interesting:

<img width="243" alt="Screenshot 2023-02-06 at 1 55 04 PM" src="https://user-images.githubusercontent.com/22692130/217060190-edd9491c-81a3-4d98-b0c1-0e3dcfb51726.png">

Sampled Backtrace:

<img width="1883" alt="Screenshot 2023-02-06 at 1 55 54 PM" src="https://user-images.githubusercontent.com/22692130/217060364-652f4f51-a4b3-4a46-b7b1-abc7670b87ec.png">

**Comment by RamblinWreck77** [comment]
From the doc link

```
The system throws an exception and generates a report when the disk writes from your app exceed a certain threshold in a 24-hour period. View the aggregated exception logs for a version of your app in the Disk Writes pane in Xcode Organizer or capture them with [MetricKit](doc://com.apple.documentation/documentation/metrickit).
```

Not super knowledgeable on SQL optimization, but the docs here seem interesting:

https://developer.apple.com/documentation/xcode/reducing-disk-writes

**Comment by jaredmixpanel** [comment]
@RamblinWreck77 I've released `v4.1.1` which adds an index and enables WAL journaling mode. I'm going to close this for now. Feel free to re-open if the issue persists.
